src/api/undeploy/helpers/delete-deployment-file.test.js

```diff
@@ -0,0 +1,31 @@
+import { featureToggles } from '~/src/helpers/feature-toggle/feature-toggles.js'
+import { isFeatureEnabled } from '~/src/helpers/feature-toggle/is-feature-enabled.js'
+import { removeDeployment } from '~/src/helpers/remove/workflows/remove-deployment.js'
+import { deleteDeploymentFiles } from '~/src/api/undeploy/helpers/delete-deployment-file.js'
+
+jest.mock('~/src/helpers/feature-toggle/is-feature-enabled')
+jest.mock('~/src/helpers/remove/workflows/remove-deployment')
```

Comment:
Quite a lot of mocking going on here. Looks like this is just testing inputs and outputs. Lots of mocks in tests for me is a smell. The code can change, the tests stay mocked and green. Real code being tested is being missed. Can the mocks be removed? Or go down to a lower level? Can the http calls be mocked with `nock`?

Reply:
It is a simple function and it's just testing if the feature toggle is respected. That is the only branching.

Overriding `process.env.???` was not really working and looked messy and I don't really want to be testing how config works in this unit test.

Reply:
Now granted there are other functions that mock too much but not this one

Reply:
Thanks for the link. I will try and use that in the future. Did not think of `config.set` as an option